### ISSUE TITLE
Client can't receive back response from server

### ISSUE THREAD

**Issue by Tanimou** [stephensanwo/fullstack-ai-chatbot#1]
I just tried the function and it appears there's a problem there:
` response = await consumer.consume_stream(stream_channel="response_channel",block=0)`
the argument "count" is missing

**Comment by stephensanwo** [comment]
Hi @Tanimou 
There is a subtle difference between the `StreamConsumer` class in the server and worker. While the worker is just waiting to read the next message (ordered queue), by setting count=1, the server is waiting for all messages and iterating over; to then send the response to the appropriate client (multiple clients connected), so we don't need the count argument.
To solve the error you are getting. Use the correct `consume_stream` methods below:

In server:
```py
async def consume_stream(self, block: int,  stream_channel):
    response = await self.redis_client.xread(
        streams={stream_channel:  '0-0'}, block=block)

    return response
```

In worker:
```py
async def consume_stream(self, count: int, block: int,  stream_channel):

    response = await self.redis_client.xread(
        streams={stream_channel:  '0-0'}, count=count, block=block)

    return response

```
Hope this helps.

**Comment by Tanimou** [comment]
So I rewrote just as what you suggested but still the same problem(see below). I'm getting confused😅
![image](https://user-images.githubusercontent.com/91100433/182035005-6c371357-6750-42f7-8fb8-6d0d4ad3497c.png)

**Comment by Scott3j** [comment]
I am running into the exact same problem. Was this ever solved by change? @Tanimou

**Comment by v-prgmr** [comment]
@Scott3j @Tanimou @stephensanwo 
for me everything works as expected while using the same stream consumer in both the server and the worker

**Comment by AMAN1620** [comment]
Yes its works. You might not be running worker/main.py. that's why its not consuming from message_stream. what you can do is run worker/main.py in other terminal and run server/main.py in other terminal. 


> I am running into the exact same problem. Was this ever solved by change? @Tanimou
